test/e2e/scenarios/initial-render/test.js

```diff
@@ -0,0 +1,128 @@
+describe("initial render", function () {
+  let mainPage
+
+  describe("for topbar", function () {
+    let topbar
+    before(function (client, done) {
+      done()
+    })
+
+    after(function (client, done) {
+      client.end(function () {
+        done()
+      })
+    })
+
+    afterEach(function (client, done) {
+      done()
+    })
+
+    beforeEach(function (client, done) {
+      mainPage = client
+        .url("localhost:3200")
```

Comment:
at first, i wasn't able to figure out where swagger-ui is rendering from.  But it looks like from here, you're running a webserver serving up swagger-ui here on port 3200?  If that's correct could you please see about having the webserver start up as part of the test (on some arbitrary port?) and hit it from there?  We'll have to also pass on the spec as a JSON object or via URL, which can be hosted by this same webserver.